lib/vm.js

```diff
@@ -39,6 +39,19 @@ const {
 // - isContext(sandbox)
 // From this we build the entire documented API.
 
+class Script extends ContextifyScript {
+  constructor(code, options) {
+    // Calling `ReThrow()` on a native TryCatch does not generate a new
+    // abort-on-uncaught-exception check. A dummy try/catch in JS land
+    // protects against that.
+    try {
+      super(code, options);
+    } catch (e) {
+      throw e; /* node-do-not-add-exception-line */
```

Comment:
Any idea for how to avoid this? Or: Is this even actually a bad thing?

Reply:
Why would it be a bad thing?

Reply:
I am not sure - it seems like something that people might find out, start to use, and be unhappy when they find out we changed/removed/etc. it

Reply:
I'm not sure what "it" refers to.  The line in the (decorated) stack trace?  What info could they glean from that?

Reply:
No, sorry, I should have been clearer - I was talking about the `node-do-not-add-exception-line` magic string and its effect of not appending the filename + arrow string in that case

Reply:
Hah, okay.  I read your question as "is it a bad thing for the throw to show up."  I wouldn't say so, and yes, I'd say it's only a matter of time before people find out about the magic comment. :-)

If you want to exclude stack frames, you can filter based on the script id.  Probably overkill for this, though.